core/templates/pages/exploration-editor-page/param-changes-editor/param-changes-editor.directive.ts

```diff
@@ -261,7 +261,7 @@ angular.module('oppia').directive('paramChangesEditor', [
               }
             };
             $scope.dragDotsImgUrl = UrlInterpolationService.getStaticImageUrl(
-              '/general/drag_dots.png');
+              '/general/drag_dots.webp');
```

Comment:
Will this file work with browsers that do not support WebP?

Reply:
add picture tag in the html file